src/main/java/executor/api/controller/ScenarioSourceController.java

```diff
@@ -0,0 +1,29 @@
+package executor.api.controller;
+
+import executor.api.model.ScenarioDTO;
+import executor.api.service.ScenarioSourceQueueHandlerImpl;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/v1/scenario")
+public class ScenarioSourceController {
+   private ScenarioSourceQueueHandlerImpl scenarioSourceQueueHandler;
+
+    public ScenarioSourceController(ScenarioSourceQueueHandlerImpl scenarioSourceQueueHandler) {
+        this.scenarioSourceQueueHandler = scenarioSourceQueueHandler;
+    }
+
+    @PostMapping("/add")
```

Comment:
you can specify an add/get/update/delete request only with an annotation without a path
```suggestion
    @PostMapping
```